### DISCUSSION TITLE
All in bootstrap, "table sort"

### DISCUSSION THREAD

**Discussion by AngelSalazarRochaEDG** [twbs/bootstrap#39789] [Q&A]
I was looking in several AI's if bootstrap currently (5.3) has a class to sort columns of a table, I don't think there is one or is there? Luckily it's not very complicated to program so for now I'll do it myself, but I also think it might be a good idea to add it to a future version of bootstrap...

**Comment by julien-deramond** [comment]
There's nothing in Bootstrap to sort the table's columns. I've found [an old comment explaining why for the JS part](https://github.com/twbs/bootstrap/issues/13804#issuecomment-47416525).
